helmfile/values/service-cluster-networkpolicy.yaml.gotmpl

```
@@ -2,22 +2,48 @@ additional:
 {{- if .Values.networkPolicies.additional }}
   {{ toYaml .Values.networkPolicies.additional | nindent 2 }}
 {{- end }}
+
+global:
+  externalLoadBalancer:
+    enabled: {{ .Values.networkPolicies.global.externalLoadBalancer.enabled }}
+    ips: {{- toYaml .Values.networkPolicies.global.externalLoadBalancer.ips | nindent 6 }}
+  ingressUsingHostNetwork: {{ .Values.ingressNginx.controller.useHostPort }}
+  apiserver:
+    ips: {{- toYaml .Values.networkPolicies.global.apiserver.ips | nindent 6 }}
+    port: {{ .Values.networkPolicies.global.apiserver.port }}
+  scNodes:
+    ips: {{- toYaml .Values.networkPolicies.global.scNodes.ips | nindent 6 }}
+  wcIngress:
+    ips: {{- toYaml .Values.networkPolicies.global.wcIngress.ips | nindent 6 }}
+  objectStorage:
+    ips: {{- toYaml .Values.networkPolicies.global.objectStorage.ips | nindent 6 }}
+    ports: {{- toYaml .Values.networkPolicies.global.objectStorage.ports | nindent 6 }}
```

Nit to keep the same order everywhere
```suggestion
  apiserver:
    ips: {{- toYaml .Values.networkPolicies.global.apiserver.ips | nindent 6 }}
    port: {{ .Values.networkPolicies.global.apiserver.port }}
  scNodes:
    ips: {{- toYaml .Values.networkPolicies.global.scNodes.ips | nindent 6 }}
  wcIngress:
    ips: {{- toYaml .Values.networkPolicies.global.wcIngress.ips | nindent 6 }}
  objectStorage:
    ips: {{- toYaml .Values.networkPolicies.global.objectStorage.ips | nindent 6 }}
    ports: {{- toYaml .Values.networkPolicies.global.objectStorage.ports | nindent 6 }}
  externalLoadBalancer:
    enabled: {{ .Values.networkPolicies.global.externalLoadBalancer.enabled }}
    ips: {{- toYaml .Values.networkPolicies.global.externalLoadBalancer.ips | nindent 6 }}
  ingressUsingHostNetwork: {{ .Values.ingressNginx.controller.useHostPort }}
```